cmd/aao/pool_test.go

```diff
@@ -0,0 +1,286 @@
+package aao
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	v1alpha1 "github.com/openshift/aws-account-operator/api/v1alpha1"
+)
+
+type MockKubeClient struct {
+	client.Client
+	mock.Mock
+}
+
+func (m *MockKubeClient) List(ctx context.Context, obj client.ObjectList, opts ...client.ListOption) error {
+	args := m.Called(ctx, obj, opts)
+	return args.Error(0)
+}
+
+func TestRun(t *testing.T) {
```

Comment:
Would you please also add the validation of the print output? you can look at the PR https://github.com/openshift/osdctl/pull/695 that is already doing it.

Reply:
Hi @petrkotas , I have added the validation of the print output in the same way i did in the PR #695. Could you please take a look?